components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -1,233 +1,199 @@
 <template>
-    <div class="mod-healthcare-professional-section">
-        <h2
-            class="mb-3.5 text-start text-primary-text text-2xl font-bold font-sans leading-normal"
-        >
-            {{ $t('modHealthcareProfessionalSection.healthcareProfessionalNameHeading') }}
-        </h2>
-        <div class="input-fields flex flex-col my-4">
-            <ModInputField
-                v-model="healthcareProfessionalSectionFields.localizedLastName"
-                data-testid="mod-healthcare-professional-section-lastName"
-                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalLastName')"
-                type="text"
-                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalLastName')"
-                :required="true"
-            />
-            <ModInputField
-                v-model="healthcareProfessionalSectionFields.localizedFirstName"
-                data-testid="mod-healthcare-professional-section-firstName"
-                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalFirstName')"
-                type="text"
-                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalFirstName')"
-                :required="true"
-            />
-            <ModInputField
-                v-model="healthcareProfessionalSectionFields.localizedMiddleName"
-                data-testid="mod-healthcare-professional-section-middleName"
-                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalMiddleName')"
-                type="text"
-                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalMiddleName')"
-                :required="false"
-            />
-            <label
-                for="name_locales"
-                class="my-2 text-primary-text text-sm font-bold font-sans"
-            >
-                {{ $t('modHealthcareProfessionalSection.labelHealthcareProfessionalNameLocale') }}
-            </label>
-            <select
-                id="name_locales"
-                v-model="healthcareProfessionalSectionFields.nameLocale"
-                data-testid="mod-healthcare-professional-section-name-locale"
-                name="Name Locales"
-                class="mb-5 px-3 py-3.5 w-96 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
-                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+    <div v-if="isHealthcareProfessionalInitialized">
```

Comment:
I think a great future issue would be to show a loader placeholder while this is being initialized! This can reduce the screen jumping all over while things are loading. 
Do you want to take a shot at writing one up for us @ShizuokaTerrier ?

Reply:
Sounds fun @ermish!